config/__init__.py

```diff
@@ -224,6 +224,7 @@ def get_variation(variation_key, variations_dict, defaults_dict):
     'type-tracking-protection-strict',
     'type-webrender-enabled',
     'type-webvr',
+    'version100',
```

Comment:
Will `version100` label be passed from the "Report site issue"? If not (i.e. it's purely based on UA detection), I wonder if it's better to not treat it as an `extra label` and instead add it on a webhook event, like we do with a browser label?

https://github.com/webcompat/webcompat.com/blob/1b72f8e76999c1aa06569d06e9ce25d2c082ff14/webcompat/webhooks/helpers.py#L164

Reply:
ah good point!
I will put the thing back on the drawing board. 

Thanks @ksy36 !